jplag.cli/src/main/java/de/jplag/CommandLineArgument.java

```diff
@@ -34,7 +27,7 @@ public enum CommandLineArgument {
     ROOT_DIRECTORY(new Builder("rootDir", String.class).nargs(NumberOfArgumentValues.ZERO_OR_MORE_VALUES)),
     NEW_DIRECTORY(new Builder("-new", String.class).nargs(NumberOfArgumentValues.ONE_OR_MORE_VALUES)),
     OLD_DIRECTORY(new Builder("-old", String.class).nargs(NumberOfArgumentValues.ONE_OR_MORE_VALUES)),
-    LANGUAGE(new Builder("-l", String.class).defaultsTo(LanguageOption.getDefault().getDisplayName()).choices(LanguageOption.getAllDisplayNames())),
+    LANGUAGE(new Builder("-l", String.class).defaultsTo(de.jplag.java.Language.SHORT_NAME).choices(LanguageLoader.getAllLanguageNames())),
```

Comment:
This `de.jplag.java.Language.SHORT_NAME` is kind of a magic String here. At least later in the `LanguageTest.java` we again use it to test the default language selection. Though there is no `LanguageOption` anymore, why not refactor this into a `getDefaultLanguageIdentifier` in the options or similar?

Reply:
I think that's a highly specific config for the CLI (JPlagOptions is defined in the core of jplag) Therefore, I would not put a method for it in JPlag.

Reply:
And what about a static method in `CommandLineArgument`?

Reply:
Yes that would be fine for me. Could also be a constant in CLA

Reply:
Fine with both :)

Reply:
EDIT: No, not possible since CLA is an Enum :(

Reply:
@JanWittler I've added a constant DEFAULT_LANGUGE to CommandLineArgument

Reply:
If we use the `identifier` proposal, maybe adjust the name then to DEFAULT_LANGUAGE_IDENTIFIER

Reply:
Agree 👍